CHANGELOG.md

```diff
@@ -48,6 +48,24 @@ _None._
 
 _None._
 
+## 3.3.0
+
+### Breaking Changes
+
+_None._
+
```

Comment:
We usually remove empty sections when releasing the version
```suggestion
```

Reply:
Thanks! Updated on 38dc91d68212eb6eeceef7a49ffee06c8191d758.